README.md

```diff
@@ -1,44 +1,114 @@
 # filebrowser-upload
 
-Upload command for https://github.com/filebrowser/filebrowser
+Upload command for <https://github.com/filebrowser/filebrowser>
 
 ## Install
 
-Download binary executable from https://github.com/guyskk/filebrowser-upload/releases
+Download binary executable from <https://github.com/guyskk/filebrowser-upload/releases> or build with
+
+```console
+foo@bar:~$ ./release.sh
+```
+
+In ```dist``` folder you will find the executable.
 
 ## Usage
 
-```
-usage: filebrowser-upload [-h] [--version VERSION] --api API --username
-                          USERNAME --password PASSWORD --dest DEST
-                          [--insecure] [--no-progress] [--override]
-                          filepath
+```console
+usage: filebrowser-upload [-h] [--version VERSION] --api API --username USERNAME 
+                          [--password PASSWORD] [--insecure] [--no-progress] 
+                          [--override] [--dry-run]
+                          {file,folder} ...
```

Comment:
Is `...` means upload multiple files or dirs?

Reply:
No, it is the same output from `-h` option